.pre-commit-config.yaml

```diff
@@ -25,6 +25,6 @@ repos:
     rev: 'v0.800'
     hooks:
     -   id: mypy
-        args: [--config-file, mypy.ini, --ignore-missing-imports, --strict]
+        args: [--config-file, mypy.ini, --ignore-missing-imports, --strict, --warn-unreachable]
```

Comment:
Not related to your change, but do we actually need `--ignore-missing-imports`? AFAIK the information about which specific imports to ignore is specified in mypy.ini.

Reply:
I can't remember if I put that there for a reason or I just copied it from somewhere else. I'd have to test it.